### PR TITLE
feat(collector): separate CollectorVersionForm and CollectorTagForm from Step Components

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
CollectorVersionForm and CollectorTagForm are isolated from the step component

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep2.vue [hunk 1]
```diff
@@ -18,25 +18,7 @@
...
-                        </p>
-                    </div>
-                </template>
-                <tags-input-group :tags="state.tags"
-                                  show-validation
-                                  :is-valid.sync="state.isTagsValid"
-                                  @update-tags="handleUpdateTags"
-                />
-            </p-field-group>
+            <collector-tag-form @updateIsTagsValid="handleChangeIsTagsValid" />
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep2.vue:21]: praise: nice separating

**Review by yuda110 — APPROVED** [review]
LGTM!

File: apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep2.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep2.vue:21]: praise: goood

**Review by WANZARGEN — APPROVED** [review]
LGTM~

**Pull request merged**